requirements.txt

```diff
@@ -1,3 +1,4 @@
+lxml
```

Comment:
Can you move this to `requirements-test.txt` please?  The reason being that we want to keep Hamilton's main dependencies nice and light.

Adding to `requirements-test.txt` will ensure the tests have the right dependencies.

Reply:
Changed the location of the module to be in the `requirements-test.txt` file. Marking this conversation as resolved.